### DISCUSSION TITLE
Porting App to Windows platform

### DISCUSSION THREAD

**Discussion by yapetrichka** [Softal-io/presentify-community#12] [Ideas]
It's be really greate if you try porting `Presentify` on Windows platform, because it's really diffucult to find analog of this app.

**Comment by rampatra** [comment]
Thanks for requesting this. If I see enough upvotes here, I can plan it.

**Comment by emailtosteveo-ai** [comment]
> It's be really greate if you try porting `Presentify` on Windows platform, because it's really diffucult to find analog of this app.

+1 to that. I know a TON of people (hundreds) who were looking at a now expired Windows analog (no longer works on Windows 10 or 11) who would move over almost en masse.
